README.md

```diff
@@ -1,2 +1,32 @@
-# FinalTaskRssParser
-For final task pull requests.
+# That's how it works
+
+* Creating rss_read class object
+* Using feedparser to get a page with function parse
+* Then using output functions get info from the page
+* Info (source link, image link, etc.) for every novelty pack in class Novelty
+* Create a pack of news filled with novelty class objects
+* When a pack of news is done come back to rss_reader.py
+* Here we prepare to output info according to arguments from console and write down information into DB
+* If there is '--to-pdf' or '--to-html' (or both arguments) argument in console we use functions 
+from PDF_and_HTML_converting to:
+        1. Get some images (to avoid many copies of pictures we first of all delete images 
+                            folder if it exists)
+        2. Add them into PDF or/and html file
+        3. Add all other information 
+
+* If there is also '--date Y%M%D' in console with '--to-pdf' or/and '--to-html' we write down into the 
```

Comment:
It would be nice to have some info about how to install your app, and also more info about the way you store news entries locally.